lucille-core/src/main/java/com/kmwllc/lucille/indexer/OpenSearchIndexer.java

```diff
@@ -121,7 +130,7 @@ protected void sendToIndex(List<Document> documents) throws Exception {
             .update((up) -> {
               up.index(index).id(docId);
               if (routingField != null) {
-                up.routing(routing);
+                up.routing(doc.getString(routingField));
```

Comment:
Is there a specific reason to remove the routing variable here?

Reply:
Not really, to be honest. Previously the routing variable was defined before checking if routingField was null, and so my thought process was that it would be cleaner(?) to only be using the routingField once we knew it wasnt null, even though the previous iteration would still work. Looking at it again we are referencing the routing variable twice so maybe it would be better to revert back by defining the routing variable. What do you think?

Reply:
It's not a huge deal either way, especially since its only used twice. I think the change you made is definitely more readable, so you're good to leave it as is. Was mostly just curious if there was a specific reason for it.